azure-pipelines-product-construction-service.yml

```diff
@@ -0,0 +1,67 @@
+# Changes the format of the Build.BuildNumber variable
+name: $(Date:yyyyMMdd)$(Rev:r)
+
+trigger:
+  branches:
+    exclude:
+      - production
```

Comment:
I don't think we want this, we want to include `main` only.
People pushing in their dev branches should not trigger build automatically. I'd say.

Reply:
yeah, you're right. If someone wants to trigger the pipeline from their dev branch, they should do so explicitly